<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BtWindow</class>
 <widget class="QWidget" name="BtWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2200</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 18pt &quot;Calibri&quot;;
background-color: rgb(200, 200, 200);
color: rgb(0, 0, 0);

 
</string>
  </property>
  <widget class="QGroupBox" name="BluetoothDeviceBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>2101</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(195, 195, 195);</string>
   </property>
   <layout class="QHBoxLayout" name="TitleBox">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QLabel" name="LableBtDevice">
      <property name="text">
       <string>Bluetooth Device: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="BtDeviceSelect"/>
    </item>
    <item>
     <widget class="QPushButton" name="SearchButton">
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ConnectButton">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="horizontalGroupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>810</y>
     <width>571</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);</string>
   </property>
   <layout class="QHBoxLayout" name="TitleBox_3">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QLabel" name="LableInputBuffer">
      <property name="text">
       <string>Input Buffer: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="InputBufferBoxTitle"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>10</y>
     <width>491</width>
     <height>55</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
font: 18pt &quot;Calibri&quot;;
text-decoration: underline;
text-decoration: underline;</string>
   </property>
   <property name="text">
    <string>EAS Oszilloscope</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="DataBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>2171</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);
background-color: rgb(195, 195, 195);</string>
   </property>
   <property name="title">
    <string>Data Telegram:</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QGroupBox" name="DataTelegramBox">
      <property name="styleSheet">
       <string notr="true"> 
 
background-color: rgb(185, 185, 185);</string>
      </property>
      <layout class="QHBoxLayout" name="TitleBox_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_DataTelegram">
         <property name="text">
          <string>Data Telegram: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_V">
         <property name="text">
          <string> 'V'</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_EntranceArea">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(195, 195, 195);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_H">
         <property name="text">
          <string> 'H'</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_N_Low">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(195, 195, 195);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_N_High">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(195, 195, 195);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_T">
         <property name="text">
          <string> 'T'</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_TriggerLow">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(195, 195, 195);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_TriggerHigh">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(195, 195, 195);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_TriggerMode">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(195, 195, 195);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_TriggerEdge">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(195, 195, 195);</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="SendButton">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="DefaultButton">
      <property name="text">
       <string>Default</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="CloseButton">
   <property name="geometry">
    <rect>
     <x>1930</x>
     <y>10</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>970</y>
     <width>221</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="pixmap">
    <pixmap>F:/Studium/7. Semster/Bachelorarbeit Android App/HAWLogo/Download.jpg</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QFrame" name="verticalFrame">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>810</y>
     <width>221</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(150, 150, 150);
color: rgb(0, 0, 0);
 </string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>Auto</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="text">
       <string>Normal</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string>Single</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>(Trigger-</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>mode)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="verticalFrame_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>840</y>
     <width>221</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(150, 150, 150);
color: rgb(0, 0, 0);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="text">
       <string>Pos</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_5">
      <property name="text">
       <string>Neg</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>(Trigger-</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>edge)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
